src/contrasts.jl

```diff
@@ -33,31 +33,38 @@ C(base = ::Any)                         # specify base level
 C(levels = ::Vector{Any}, base = ::Any) # specify levels and base
 ```
 
-If specified, levels will be checked against data when generating a
-`ContrastsMatrix`. Any mismatch will result in an error, because missing data
-levels would lead to empty columns in the model matrix, and missing contrast
-levels would lead to empty or undefined rows.
+**Arguments**
 
-You can also specify the base level of the contrasts. The actual interpretation
-of this depends on the particular contrast type, but in general it can be
-thought of as a "reference" level.  It defaults to the first level.
+* `levels`: Optionally, the data levels can be specified here.  This allows you
+  to specify the order of the levels.  If specified, the levels will be checked
```

Comment:
"the levels provided here will be checked against the levels actually present in the data"?